db/schema.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
-
 # This file is auto-generated from the current state of the database. Instead
```

Comment:
Request for change - Did you run rubocop on the project before submitting the PR?

Reply:
I did. Strange.. I dont see that on my local.

Reply:
I did. I updated the rubocop configurations and I didn't need it anymore.